examples/flutter_gallery/pubspec.yaml

```diff
@@ -10,7 +10,8 @@ dependencies:
     path: ../../packages/flutter_markdown
   # Also update dev/benchmarks/complex_layout/pubspec.yaml
   # and examples/flutter_gallery/pubspec.yaml
-  flutter_gallery_assets: '0.0.21'
+  flutter_gallery_assets:
+    git: https://flutter.googlesource.com/gallery-assets
```

Comment:
Can you add a git commit hash here so that the version is pinned when we go back through history ?